public/utils/validate.js

```diff
@@ -30,6 +30,13 @@ export const validateActionName = trigger => value => {
   if (matches.length > 1) return 'Action name is already used';
 };
 
+export const validateActionThrottle = action => value => {
+  if (_.get(action, 'throttle_enabled')) {
+    if (!value) return 'Need to set throttle value';
```

Comment:
Recommend changing this; if they put in 0 we should not give an error saying 'Need to set throttle vaue'
They did set it, it's just an incorrect value.

Reply:
I think we can just make this better, saying `Throttle should be positive number` ? so either in case of blank , or invalid message stays same.

Reply:
Hi, to make to code logic simple, I will adopt Mihir's suggestion. Use more generic message to cover both cases.

Reply:
I'm still not a fan of the solution; if you enter 0 it'll say:
'Throttle should be positive number'
If you enter -1 it'll then change to:
'Must be a positive integer'

If we want to simplify the logic then let's just remove the if(!value) completely and just return the validateInterval so the message is actually the same. Any null,undefined,etc. values will fail the Number.isInteger check

Reply:
Also why is it called validateInterval? That might of been a validation function for a specific use case on another input. If we're reusing it then we should rename this function to be generic. Check where else it is used.

Reply:
offline discussed, rename the function validateInterval to validatePositiveInteger